x/feemarket/spec/01_concepts.md

```diff
@@ -4,33 +4,60 @@ order: 1
+
+Before EIP-1559 the transaction fee is calculated with
+
+```
+fee = gasPrice * gasLimit
+```
+
+, where `gasPrice` is the price per gas and `gasLimit` describes the amount of gas required to perform the transaction. The more complex operations a transaction requires, the higher the gasLimit (See [Executing EVM bytecode](https://docs.evmos.org/modules/evm/01_concepts.html#executing-evm-bytecode)). To submit a transaction, the signer needs to specify the `gasPrice`.
+
+With EIP-1559 enabled, the transaction fee is calculated with
+
+```
+fee = (baseFee + priorityTip) * gasLimit
+```
+
+, where `baseFee` is the fixed-per-block network fee per gas and `priorityTip` is an additional fee per gas that can be set optionally. Note, that both the base fee and the priority tip are a gas prices. To submit a transaction with EIP-1559, the signer needs to specify the `gasFeeCap` a maximum fee per gas they are willing to pay total and optionally the `priorityTip` , which covers both the priority fee and the block's network fee per gas (aka: base fee).
+
 ## Base fee
 
-The base fee is a global base fee defined at the consensus level. It is adjusted for each block based on the total gas used in the previous block and gas target (block gas limit divided by elasticity multiplier)
+The base fee per gas is a global gas price defined at the consensus level. It is tored as a module parameter and is adjusted at the end of each block based on the total gas used in the previous block and gas target (`block gas limit / elasticity multiplier`):
 
-- it increases when blocks are above the gas target
-- it decreases when blocks are below the gas target
+- it increases when blocks are above the gas target,
+- it decreases when blocks are below the gas target.
 
-Unlike the Cosmos SDK local `minimal-gas-prices`, this value is stored as a module parameter which provides a reliable value for validators to agree upon.
+Instead of burning the base fee (as implemented on Ethereum), the `feemarket` module allocates the base fee for regular [Cosmos SDK fee distribution](https://docs.cosmos.network/master/modules/distribution/).
 
 ## Tip
 
-In EIP-1559, the `tip` is a value that can be added to the `baseFee` in order to incentive transaction prioritization.
+In EIP-1559, the `max_priority_fee_per_gas`, often referred to as `tip`, is an additional gas price that can be added to the `baseFee` in order to incentive transaction prioritization. The higher the tip, the more likely the transaction is included in the block.
+
+In the Cosmos SDK, however, there is no notion of prioritization. Thus the tip for an EIP-1559 transaction on Ethermint should be zero (`MaxPriorityFeePerGas` JSON-RPC endpoint returns `0`).
```

Comment:
```suggestion
Until the Cosmos SDK version v0.46, however, there is no notion of transaction prioritization. Thus the tip for an EIP-1559 transaction on Ethermint should be zero (`MaxPriorityFeePerGas` JSON-RPC endpoint returns `0`).
```

Reply:
we should also link to the ADR that allows for prioritization

https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-067-mempool-refactor.md